src/commonMain/kotlin/com/algolia/search/serialize/KeysTwo.kt

```diff
@@ -187,3 +187,6 @@ public const val KeyMatchAlternatives = "match.alternatives"
 public const val KeyTypes = "types"
 public const val KeyPending = "pending"
 public const val KeyGetClusters = "getClusters"
+public const val KeyEU = "eu"
```

Comment:
@Ant-hem @aseure Can you confirm is EU here? Or is like analytics here should be DE?

Reply:
Yep It can be `EU` or `US` according to : https://www.algolia.com/doc/rest-api/recommendation/

```
You can use one of the following domains:
- United States: https://recommendation.us.algolia.com
-  Europe: https://recommendation.eu.algolia.com
```

Reply:
Thanks for the confirmation!